scripts/configuration/BuildGroup.cake

```diff
@@ -70,8 +87,44 @@ public class BuildGroup
             {
                 var platform = childNode.Attributes.GetNamedItem("platform")?.Value;
                 var configuration = childNode.Attributes.GetNamedItem("configuration")?.Value;
-                _builds.Add(new BuildConfig(platform, configuration));
+                _builds.Add(new BuildConfig(platform, configuration, _vs2019path));
+            }
+        }
+    }
+
+    // Copy from the cake's internal MSBuildRunner.GetVisualStudio2019Path()
+    private string GetVisualStudio2019Path()
```

Comment:
We need to follow the process of OSS attribution if we copy some Cake source file here.